powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/entity/fido2/AttestationObject.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2023 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.security.powerauth.client.model.entity.fido2;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+
+/**
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Data
+public class AttestationObject {
+
+    @JsonIgnore
+    private String encoded;
+
+    private String fmt;
+
+    private AuthenticatorData authData;
+    private AttestationStatement attStmt;
```

Comment:
Is this our API or does it come from specification? I am asking whether we may avoid abbreviations.

Reply:
This corresponds to the specification, see e.g. https://www.w3.org/TR/webauthn-2/#sctn-generating-an-attestation-object